binary-build-list.json

```diff
@@ -0,0 +1,11 @@
+[
+    "sui",
+    "sui-node",
+    "sui-tool",
+    "sui-faucet",
+    "sui-test-validator",
+    "sui-data-ingestion",
+    "sui-bridge",
+    "sui-bridge-cli",
+    "sui-graphql-rpc"
```

Comment:
This doesn't match the list in `sui-release-binaries.yaml` btw

Reply:
Where did you find this file?

Reply:
Yes, cause that workflow is more for internal usage, and the original list here is more for public consumption.

Reply:
Like `stress`, `sui-proxy` , `sui-metric-checker`, `sui-analytics-indexer` and `sui-security-watchdog` binaries. But maybe we should package them all together for public consumption as well?

Reply:
Nah i don't think those need to be added to the public binaries, but I do think we should have a second `binary-build-list.json` which `sui-release-binaries.yaml` uses

Reply:
Yep, that's coming next. I am just doing this on the sui public repo right now